Palace/Accounts/Library/AccountsManager.swift

```diff
@@ -24,8 +24,7 @@ let currentAccountIdentifierKey  = "TPPCurrentAccountIdentifier"
 
   static let TPPNationalAccountUUIDs = [
     "urn:uuid:6b849570-070f-43b4-9dcc-7ebb4bca292e", //DPLA
-    "urn:uuid:f60b644e-4955-4996-a4e5-a192feb4e7f8", //Internet Archive
-    "urn:uuid:5278562c-d642-4fda-ad7e-1613077cfb8d", //Open Textbook Library
+    "urn:uuid:f60b644e-4955-4996-a4e5-a192feb4e7f8"  //Internet Archive
```

Comment:
@vladimirfedorov We should remove the "Internet Archive" entry here, as well, since it has been removed from the registry.

Additionally, it may be helpful to change the comment on the "DPLA" entry to "Palace Bookshelf" for easier identification in the future, since that's what we use in branding materials and in the registry.